model_server_util_tools/model_packaging/export_model_utils.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+Helper utils for Model Export tool
+"""
+
+import os
+import json
+import zipfile
+from model_server_util_tools.model_packaging.model_packaging_error_codes import ModelPackagingErrorCodes
+from model_server_util_tools.model_packaging.manifest_components.publisher import Publisher
+from model_server_util_tools.model_packaging.manifest_components.manifest import Manifest
+from model_server_util_tools.model_packaging.manifest_components.engine import Engine
+from model_server_util_tools.model_packaging.manifest_components.model import Model
+from model_server_util_tools.model_packaging.model_packaging_error import ModelPackagingError
+from model_server_util_tools import model_packaging
+
+MODEL_ARCHIVE_EXTENSION = '.mar'
+MODEL_SERVER_VERSION = '1.0'
+MODEL_ARCHIVE_VERSION = model_packaging.__version__
+MANIFEST_FILE_NAME = 'MANIFEST.json'
+MAR_INF = 'MAR-INF'
+ONNX_TYPE = '.onnx'
+
+
+class ModelExportUtils(object):
+    """
+    Helper utils for Model Export tool.
+    This class lists out all the methods such as validations for model archiving, ONNX model checking etc.
+    This is to keep the code in export_model.py clean and simple.
+    """
+
+    @staticmethod
+    def check_mar_already_exists(model_name, export_file):
+        """
+        Function to check if .mar already exists
+        :param model_name:
+        :param export_file:
+        :param model_archive_extension:
+        :return:
+        """
+        if export_file is None:
+            export_file = os.path.join(os.getcwd(), '{}{}'.format(model_name, MODEL_ARCHIVE_EXTENSION))
+
+        if os.path.exists(export_file):
+            raise ModelPackagingError(ModelPackagingErrorCodes.MODEL_ARCHIVE_ALREADY_PRESENT,
```

Comment:
We should provide an option to force overwrite

Reply:
Done. Handled in : ab6a918fdd4745c8ccaf884409a7393163a7ddb9